### DISCUSSION TITLE
Noob question: Importing to Eclipse using Gradle.

### DISCUSSION THREAD

**Discussion by smcd0** [deepjavalibrary/djl#2069] [Q&A]
I've been trying to get going with the online tutorial. I would prefer to work within my Eclipse IDE rather than a Jupyter Notebook. I am fairly new to Java. I have downloaded the djl library. I navigate to File>Import>Gradle>Existing Gradle Project (In the Eclipse IDE) and then type in the path "C:\Users\samuel\djl-0.19.0". 

This imports all the files into my project explorer, but I'm not particularly sure what to actually do with them at this point... I'd like to start working through chapter 2.1, which uses the ndarray method. I can't seem to access this method. The following lines of code:

`NDManager manager = NDManager.newBaseManager();`
`var x = manager.arange(12);`
`x`

will not work, because NDManager is not found. 

In python, the equivalent is just using pip to install a package and then importing the package... What is the Java/DJL equivalent?

**Comment by frankfliu** [comment]
In Java, you also need import package, you can find javadoc regarding [core DJL API](https://javadoc.io/doc/ai.djl/api/latest/index.html)

For jupyter notebook are you referring to, the java classes is imported in:
```
%load ../utils/djl-imports
```

You can find the source here: https://github.com/deepjavalibrary/d2l-java/blob/master/utils/djl-imports.ipynb
